src/main/java/com/supermartijn642/movingelevators/MovingElevators.java

```diff
@@ -29,12 +32,16 @@
     public static DisplayBlock display_block;
     @GameRegistry.ObjectHolder("movingelevators:button_block")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
The lines you referenced seem to be fine.
Only the variable name doesn't match the naming conventions, but that's the same way I have it since its the same as the block's registry id.

You also talk about methods, but you haven't referenced any methods. Which methods are you talking about?